server/tests/src/test/java/org/infinispan/server/persistence/ManagedConnectionOperations.java

```diff
@@ -0,0 +1,95 @@
+package org.infinispan.server.persistence;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.persistence.jdbc.configuration.JdbcStringBasedStoreConfigurationBuilder;
+import org.infinispan.server.test.core.category.Persistence;
+import org.infinispan.server.test.junit4.DatabaseServerRule;
+import org.infinispan.server.test.junit4.InfinispanServerRule;
+import org.infinispan.server.test.junit4.InfinispanServerTestMethodRule;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(Persistence.class)
+public class ManagedConnectionOperations {
+
+   @ClassRule
+   public static InfinispanServerRule SERVERS = PersistenceIT.SERVERS;
+
+   @ClassRule
+   public static DatabaseServerRule DATABASE = new DatabaseServerRule(SERVERS);
+
+   @Rule
+   public InfinispanServerTestMethodRule SERVER_TEST = new InfinispanServerTestMethodRule(SERVERS);
+
+   public ManagedConnectionOperations() {
+      DATABASE.setDatabaseType("h2");
+   }
+
+   private org.infinispan.configuration.cache.ConfigurationBuilder createConfigurationBuilder() {
+
+      org.infinispan.configuration.cache.ConfigurationBuilder builder = new org.infinispan.configuration.cache.ConfigurationBuilder();
+      builder.persistence().addStore(JdbcStringBasedStoreConfigurationBuilder.class)
+            .fetchPersistentState(false)
+            .ignoreModifications(false)
+            .purgeOnStartup(false)
+            .shared(false)
+            .table()
+            .dropOnExit(true)
+            .createOnStart(true)
+            .tableNamePrefix("TBL")
```

Comment:
The following can be removed as they're just using the default values:

- fetchPersistentState
- ignoreModifications
- ignoreModifications
- shared

Reply:
Done